shared/templates/template_ANSIBLE_grub2_bootloader_argument

```diff
@@ -29,7 +29,14 @@
 
 {{% else %}}
 
-- name: update bootloader menu
-  shell: '/usr/bin/grub2-editenv - set "$(grub2-editenv - list | grep kernelopts) {{{ ARG_NAME_VALUE }}}"'
+- name: get current kernel parameters
+  command: 'grep kernelopts `/usr/bin/grub2-editenv - list`'
+  register: kernelopts
+
+- set_fact:
+  kerneloptsfact: "{{ kernelopts.stdout }}"
+
+- name: Update the bootloader menu
+  command: '/usr/bin/grub2-editenv - set {{ kerneloptsfact }} {{{ ARG_NAME_VALUE }}}'
```

Comment:
`command: /usr/bin/grub2-editenv set {{ item }} {{{ ARG_NAME_VALUE }}}`
 
Also need to add:
````
with_items: "{{ kernelopts.stdout_lines | select('match', '^kernelopts.*') | list }}"
when:
  - kernelopts.stdout_lines is defined
  - kernelopts.stdout_lines | length > 0
````

Reply:
Thank you @redhatrises I am not very familiar with Ansible so this was a nice introduction.

Reply:
@vojtapolasek yeah no problem. One question about `grub2-editenv` that I have is the dash supposed to exist like `/usr/bin/grub2-editenv - set` or is it supposed to be `/usr/bin/grub2-editenv set` without the dash?

Reply:
The dash needst to be there, without it it asks for the target file location and the dash is also used in the rhel8 documentation.